x-pack/plugins/fleet/common/constants/epm.ts

```diff
@@ -42,17 +36,11 @@ in their custom policy editor implementations.
 
 */
 
-export const unremovablePackages = [
+export const autoUpdatePackages = [
   FLEET_SYSTEM_PACKAGE,
   FLEET_ELASTIC_AGENT_PACKAGE,
   FLEET_SERVER_PACKAGE,
```

Comment:
I don't think these 3 should be in auto update. Only the 3 we had before: endpoint, apm, synthetics. This will likely change in the future, but needs to happen with some other related changes.